packages/moonstone/styles/variables.less

```diff
@@ -256,8 +256,6 @@
 @moon-input-padding: 0 @moon-input-icon-spacing;
 @moon-input-font-weight: 600;
 @moon-input-text-indent: 0.2em;
-@moon-input-disabled-opacity: 0.3;
```

Comment:
I'm hesitant to move these variables right now since it would be a potential breaking change which I'd like to avoid in an RC. I think we're safe to leave these here because the values in `colors(-light).less` will override them via `.applySkins()`.